app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -116,16 +116,16 @@ def save_container_limits_inventory(ems, hashes, target = nil)
     store_ids_for_new_records(ems.container_limits, hashes, :ems_ref)
   end
 
-  def save_container_limit_items_inventory(container_limit, hashes, target = nil)
+  def save_container_limit_items_inventory(container_entity, hashes, target = nil)
     return if hashes.nil?
-    container_limit.container_limit_items.reset
+    container_entity.container_limit_items.reset
     deletes = if target.kind_of?(ExtManagementSystem)
                 :use_association
               else
                 []
               end
-    save_inventory_multi(container_limit.container_limit_items, hashes, deletes, [:resource, :item_type])
-    store_ids_for_new_records(container_limit.container_limit_items, hashes, [:resource, :item_type])
+    save_inventory_multi(container_entity.container_limit_items, hashes, deletes, [:resource, :item_type])
+    store_ids_for_new_records(container_entity.container_limit_items, hashes, [:resource, :item_type])
```

Comment:
Just to make sure: will these do the right thing given limit items with same (container_entity_id, resource, item_type) but different container_entity_type?
I suppose yes, as each call deals with one `container_entity.container_limit_items` association, constrained to specific id & type.